src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2460,6 +2487,14 @@ void processContextElement
       searchContextProviders(tenant, servicePathV, *enP, ceP->contextAttributeVector, cerP);
       cerP->statusCode.fill(SccOk);
       responseP->contextElementResponseVector.push_back(cerP);
+
+      //
+      // If no context providers found, then the UPDATE was simply for a non-found entity and an error should be returned
+      //
+      if (forwardsPending(responseP) == false)
```

Comment:
I understand that at the end we disabled v2 forwarding in the upper layer (serviceRoutine), thus this check probably is not needed at mongoBackend and we could go back to the situation in which unit test worked without modifications.